src/Resources/views/CRUD/edit_mongo_association_script.html.twig

```diff
@@ -16,6 +16,8 @@ This code manage the many-to-[one|many] association field popup
 
 #}
 
+{% sonata_template_deprecate '@SonataAdminBundle/CRUD/Association/edit_many_script.html.twig' %}
```

Comment:
Do we still need the `Bundle` part in `SonataAdminBundle` when refering to the new twig template path syntax?

Reply:
@core23 I think we do not need it, did anyone test if this transition break something?